src/Deck.java

```diff
@@ -0,0 +1,69 @@
+import java.util.Random;
+
+
+public class Deck {
+
+    private Card[] deck;
+    private int currentCard; //index of next card to be dealt
+
+    public Deck() {
+
+        String[] faces = {"2", "3", "4", "5", "6", "7", "8", "9", "10", "Jack", "Queen", "King", "Ace"};
+        String[] suits = {"Spades", "Clubs", "Diamonds", "Hearts"};
+
+        deck = new Card[52];
+        currentCard = 0;
+
+        /**
+         * Building the deck
+         */
+        int suitCounter = 0;
+        while (suitCounter < 4) {
+            int faceNum = 0;
+            while (faceNum < 13) {
+                deck[(faceNum + (suitCounter * 13))] = new Card (
+                        suits[suitCounter],
+                        faces[faceNum],
+                        faceNum+2);
+
+                faceNum++;
+            }
+            suitCounter++;
+        }
+    }
+
+    /**
+     * Printing the deck
+     */
+    public void displayDeck() {
+
+        for (Card card : deck)
+            System.out.println(card);
```

Comment:
what is the console output for this ?
should it be  `System.out.println(card.toString()); ` ?

Reply:
everything worked fine for me, but it might be good practice for me to use the ".toString" 

my output was 52 different cards 
"Jack of Hearts
8 of Spades
...
etc."